bears/scss/SCSSLintBear.py

```diff
@@ -21,4 +23,265 @@ class SCSSLintBear:
+        :param check_pseudo_elements:
+            Pseudo-elements, like `::before`, and `::first-letter`, should be
+            declared with two colons. Pseudo-classes, like `:hover` and
+            `:first-child`, should be declared with one colon.
+
+            ```
+            p::before {
+              content: '>'
+            }
+
+            p:hover {
+              color: red;
+            }
+            ```
+        '''
+
+        options = {'BangFormat': {'enabled': check_bang_format,
+                                  'space_before_bang': space_before_bang,
+                                  'space_after_bang': space_after_bang},
+                   'ChainedClasses': {'enabled': not allow_chained_classes},
+                   'ColorKeyword': {'enabled': not allow_color_keywords},
+                   'ColorVariable': {'enabled': use_color_variables},
+                   'DebugStatement': {'enabled': not allow_debug_statement},
+                   'DeclarationOrder': {'enabled': check_declaration_order},
+                   'DuplicateProperty': {
+                       'enabled': not allow_duplicate_properties,
+                       'ignore_consecutive':
+                           allow_consecutives_duplicate_property},
+                   'ElsePlacement': {'enabled': True,
+                                     'style': else_placement_style},
+                   'EmptyLineBetweenBlocks': {
+                       'enabled': force_empty_line_between_blocks,
+                       'ignore_single_line_blocks': True},
+                   'EmptyRule': {'enabled': not allow_empty_rules},
+                   'HexLength': {'enabled': True,
+                                 'style': hexadecimal_length_style},
+                   'HexNotation': {'enabled': True,
+                                   'style': hexadecimal_notation_case},
+                   'HexValidation': {'enabled': allow_hexadecimal_validation},
+                   'IdSelector': {'enabled': not allow_id_selector},
+                   'ImportantRule': {'enabled':
+                                     not allow_important_rule_in_properties},
+                   'Indentation': {'enabled': True,
+                                   'allow_non_nested_indentation': False,
+                                   'character': ('space'
+                                                 if use_spaces else 'tab'),
+                                   'width': indent_size},
+                   'LeadingZero': {'enabled': True,
+                                   'style': leading_zero},
+                   'MergeableSelector': {'enabled': allow_mergeable_selectors,
+                                         'force_nesting': True},
+                   'NestingDepth': {'enabled': True,
+                                    'max_depth': max_depth,
+                                    'ignore_parent_selectors': False},
+                   'NameFormat': {'enabled': True,
```

Comment:
Another way of writing this is:
```
 +                  'NameFormat': {
 +                      'enabled': True,
 +                      'allow_leading_underscore': allow_leading_underscore,
 +                      'function_convention': '_'.join(
 +                          function_naming_convention.split()),
 +                      'mixin_convention': '_'.join(
 +                          mixin_naming_convention.split()),
 +                      'variable_convention': '_'.join(
 +                          variable_naming_convention.split()),
 +                      'placeholder_convention': '_'.join(
 +                          placeholder_naming_convention.split())},
```
Which IMO is a bit more readable. But this is ok too.